public/toolkit/styles/molecules/_inputs.scss

```diff
@@ -383,3 +383,26 @@ input.icon-input-field {
   line-height: $icon-input-height;
   height: $icon-input-height;
 }
+
+// Search text field with x icon to clear
+// Javascript must be added to
+$clear-icon-margin: 13.5px;
+$clear-icon-height: 16px;
+.search {
+  display: inline-block;
+  position: relative;
+
+  input {
+    width: 15.5rem;
```

Comment:
Shouldn't the width just be 100% of the parent? That way it works the same as a button or w/e where the column width it's contained in decides the element width